src/main/kotlin/no/nav/familie/ef/sak/behandling/fremleggsoppgave/FremleggsoppgaveController.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.familie.ef.sak.behandling.fremleggsoppgave
+
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+@RestController
+@RequestMapping("/api/fremleggsoppgave")
+@ProtectedWithClaims(issuer = "azuread")
+class FremleggsoppgaveController(private val fremleggspppgaveService: FremleggsoppgaveService) {
+
+    @GetMapping("/{behandlingid}")
+    fun hentFremleggsoppgave(@PathVariable behandlingid: UUID): Ressurs<FremleggsoppgaveDto?> {
+        val fremleggsOppgave = fremleggspppgaveService.hentFremleggsoppgave(behandlingid)
```

Comment:
```suggestion
        val fremleggsOppgaveEllerNull = fremleggspppgaveService.hentFremleggsoppgaveEllerNull(behandlingid)
```
Er vel egentlig dette som blir gjort?

Reply:
Joda, men typen er nullable i signaturen og.